server/src-lib/Hasura/Tracing.hs

```diff
@@ -36,6 +36,9 @@ import qualified Network.HTTP.Client         as HTTP
 import qualified Network.HTTP.Types.Header   as HTTP
 import qualified System.Random               as Rand
 import qualified Web.HttpApiData             as HTTP
+import qualified Data.Binary                 as BIN
```

Comment:
Small code style thing: the idiomatic way to do a qualified import here, if you want to call it "bin", would be `import qualified Data.Binary as Bin`. When I first read the code I was trying to think what module could possibly satisfy `B*.I*.N*` :sweat_smile:

Reply:
Fixed it, Thanks :)